zqd/api/api.go

```diff
@@ -112,9 +112,15 @@ type PacketPostStatus struct {
 
 type LogPostRequest struct {
 	Paths          []string             `json:"paths"`
+	StopErr        bool                 `json:"stop_err"`
```

Comment:
I feel like this deserves a more intention-revealing name, though I'm not sure what. FailOnLogError? FailOnInputError? SomethingElse?

Reply:
Brevity vs explicitness tradeoff... I used StopErr here to be consistent with the `zq` command line flag. I think we should change both jointly if we want to use something else.

Reply:
Let's change 'em later, then.